src/types/interactions/application_commands/application_command_callback_data.ts

```diff
@@ -0,0 +1,21 @@
+import { Embed } from "../../embeds/embed.ts";
+import { AllowedMentions } from "../../messages/allowed_mentions.ts";
+import { SnakeCaseProps } from "../../util.ts";
+
+export interface ApplicationCommandInteractionData {
+  /** Is the response TTS */
+  tts?: boolean;
+  /** Message Content */
+  content?: string;
+  /** Supports up to 10 embeds */
+  embeds?: Embed[];
+  /** Allowed Mentions object */
+  allowedMentions?: AllowedMentions;
+  /** Set to `64` to make your response ephemeral */
+  flags?: number;
+}
```

Comment:
![Screenshot_20210329_162137](https://user-images.githubusercontent.com/34645569/112835823-d9550580-90aa-11eb-8a57-3aa7798ca419.png)
Is this correct?

Reply:
wrong interface name

Reply:
See the file name :roll_eyes:

Reply:
Somehow my git didn't push right